README.md

```diff
@@ -1,2 +1,55 @@
 # err-topic-a-day
-An errbot plugin to post a topic a day to a channel for discussion
+
+## Description
+This Bot (http://errbot.io) posts to slack  plugin to post a topic a day to a channel for discussion
+
+## Usage
+TODO: FILL THIS
```

Comment:
Could link to the errbot plugin install docs

Reply:
Done, but not in usage, This code works by running the python module only? if so, I can just add this command here

Reply:
This code is a plugin for errbot. It is only run as a part of an errbot installation. You can link to here: https://errbot.readthedocs.io/en/latest/user_guide/administration.html#installing-plugins